src/editors/sharedComponents/TinyMceWidget/hooks.js

```diff
@@ -57,12 +57,18 @@ export const replaceStaticwithAsset = ({
   imageSrcs.forEach(src => {
     const currentContent = content;
     let staticFullUrl;
-    if (src.startsWith('/static/') && imageUrls.length > 0) {
-      const imgName = src.substring(8, src.indexOf('"'));
+    const isStatic = src.startsWith('/static/');
+    const isExpandableAsset = src.startsWith('/assets/') && editorType === 'expandable';
```

Comment:
Can you explain the logic of what you are doing here in words? I am really bad at reading regex

Reply:
I am removing everything except the image name. I make sure that the replacement starts at the begin file path and matches all characters until the last slash/